inst/tests/tests.Rraw

```diff
@@ -21141,3 +21141,13 @@ dt = data.table(id = 1:25)
 test(2314.1, any(grepl("<int>", tail(capture.output(print(dt, class = TRUE)), 2))), TRUE)
 # Test that class=TRUE with col.names="top" doesn't show classes at bottom
 test(2314.2, !any(grepl("<int>", tail(capture.output(print(dt, class = TRUE, col.names = "top")), 2))), TRUE)
+
+# Test that tzone behavior stops warning with ignore_tzone=TRUE
+tm1 = .POSIXct(1, "UTC")
+dt1 = data.table(a = .POSIXct(0), b = .POSIXct(2, "UTC"))
```

Comment:
nit style comments:

 - no space around `=` in keyword args
 - use `=` for assignment
 
 (I'd offer suggested changes but it doesn't work in GitHub on the huge tests.Rraw file)